actions/cloud.tsx

```diff
@@ -1,6 +1,6 @@
-
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
+
```

Comment:
The old version of `eslint-plugin-header` accepted whitespace above the header. The new one does not.